developers/academy/js/starter_multimodal_data/_snippets/101_connect.mts

```diff
@@ -0,0 +1,171 @@
+      documentationHref: 'https://docs.cohere.com/reference/chat',
+      name: 'Generative Search - Cohere'
+    },
+    'generative-mistral': {
+      documentationHref: 'https://docs.mistral.ai/api/',
+      name: 'Generative Search - Mistral'
+    },
+    'generative-openai': {
+      documentationHref: 'https://platform.openai.com/docs/api-reference/completions',
+      name: 'Generative Search - OpenAI'
+    },
+    'generative-google': {
+      documentationHref: 'https://cloud.google.com/vertex-ai/docs/generative-ai/chat/test-chat-prompts',
+      name: 'Generative Search - Google'
+    },
+    'multi2vec-google': {
+      documentationHref: 'https://cloud.google.com/vertex-ai/generative-ai/docs/embeddings/get-multimodal-embeddings',
+      name: 'Google Multimodal Module'
+    },
+    'qna-openai': {
+      documentationHref: 'https://platform.openai.com/docs/api-reference/completions',
+      name: 'OpenAI Question & Answering Module'
+    },
+    'ref2vec-centroid': {},
+    'reranker-cohere': {
+      documentationHref: 'https://txt.cohere.com/rerank/',
+      name: 'Reranker - Cohere'
+    },
+    'reranker-voyageai': {
+      documentationHref: 'https://docs.voyageai.com/reference/reranker-api',
+      name: 'Reranker - VoyageAI'
+    },
+    'text2vec-aws': {
+      documentationHref: 'https://docs.aws.amazon.com/bedrock/latest/userguide/titan-embedding-models.html',
+      name: 'AWS Module'
+    },
+    'text2vec-cohere': {
+      documentationHref: 'https://docs.cohere.ai/embedding-wiki/',
+      name: 'Cohere Module'
+    },
+    'text2vec-huggingface': {
+      documentationHref: 'https://huggingface.co/docs/api-inference/detailed_parameters#feature-extraction-task',
+      name: 'Hugging Face Module'
+    },
+    'text2vec-jinaai': {
+      documentationHref: 'https://jina.ai/embeddings/',
+      name: 'JinaAI Module'
+    },
+    'text2vec-openai': {
+      documentationHref: 'https://platform.openai.com/docs/guides/embeddings/what-are-embeddings',
+      name: 'OpenAI Module'
+    },
+    'text2vec-google': {
+      documentationHref: 'https://cloud.google.com/vertex-ai/docs/generative-ai/embeddings/get-text-embeddings',
+      name: 'Google Module'
+    },
+    'text2vec-voyageai': {
+      documentationHref: 'https://docs.voyageai.com/docs/embeddings',
+      name: 'VoyageAI Module'
+    }
+  },
+  version: '1.25.5'
+}
+// END OutputGetMeta
+
+
+
+client.close()
+
+client = await weaviate.connectToWeaviateCloud(
+  process.env.WCD_URL as string,
+  {
+    authCredentials: new weaviate.ApiKey(process.env.WCD_API_KEY as string),
+    headers: {
+      'X-Cohere-Api-Key': process.env.COHERE_APIKEY as string,  // Replace with your inference API key
+    }
+  }
+)
+
+// TryFinallyCloseDemo
+
+// Instantiate your client (not shown). e.g.:
+// client = weaviate.connectToWeaviateCloud(...) or
+// client = weaviate.connectToLocal(...)
+
+try {
+  // Work with the client here 
+  if (await client.isLive()) {
+    // ...
```

Comment:
It is not super clear to me what code should go in this block, vs below/outside of the block. 

If the server is not live / we can't connect to it, it may not be worth proceeding as you can't do anything.  

I would just keep it simple and use an assert personally. wdyt?

Reply:
I changed the position of this. 
Did you look at this only through the snippet file or on the academy page? there is a line explaining it.